pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -82,7 +83,7 @@ func RunClusterInfo(factory *cmdutil.Factory, out io.Writer, cmd *cobra.Command)
 					link += "http://" + ip + ":" + strconv.Itoa(port.Port) + " "
 				}
 			} else {
-				link = client.Host + "/api/v1beta3/proxy/namespaces/" + service.ObjectMeta.Namespace + "/services/" + service.ObjectMeta.Name
+				link = client.Host + "/api/" + latest.Version + "/proxy/namespaces/" + service.ObjectMeta.Namespace + "/services/" + service.ObjectMeta.Name
```

Comment:
I think it's still better than hard coding v1, because at least we will have a correct URL when we update to a new API but keep the URL schema unchanged.